data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json

```
@@ -0,0 +1,35 @@
+[
+  {
+    "id": "gaiaschosen_samsara",
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You let your mana spin in the world's life flow, renewing your body as if prematurely reincarnated.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 250000,
+    "final_casting_time": 50000,
+    "casting_time_increment": -6250,
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "components": "spell_components_samsara",
+    "difficulty": 8,
+    "min_damage": -20,
+    "max_damage": -200,
+    "damage_increment": -4.5,
+    "max_level": 40,
+    "extra_effects": [ { "id": "gaiaschosen_samsara_cureall", "hit_self": true } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ]
```

```suggestion
    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "LOUD" ]
```
this isn't really what this flag's for